easybuild/easyconfigs/z/zsh/zsh-5.9-GCCcore-13.3.0.eb

```diff
@@ -0,0 +1,30 @@
+easyblock = 'ConfigureMake'
+
+name = 'zsh'
+
+version = '5.9'
+
+homepage = 'http://www.zsh.org/'
+description = """
+Zsh is a shell designed for interactive use, although it is also a powerful scripting language.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '13.3.0'}
+
+source_urls = ['http://prdownloads.sourceforge.net/%(namelower)s']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['9b8d1ecedd5b5e81fbf1918e876752a7dd948e05c1a0dba10ab863842d45acd5']
+
+builddependencies = [
+    ('binutils', '2.42'),
+]
+
+dependencies = [
+    ('ncurses', '6.5'),
+]
+
```

Comment:
sanity_check_paths is missing here.
can you also add sanity_check_commands?
it' probably the same as in zsh-5.9.eb:
```
sanity_check_paths = {
    'files': ['bin/zsh'],
    'dirs': ['lib/zsh/%(version)s', 'share'],
}

sanity_check_commands = ['zsh --version']
```

Reply:
Oh, yes, sorry.
It is now added.